lib/ex_unit/lib/ex_unit/case.ex

```diff
@@ -389,6 +393,10 @@ defmodule ExUnit.Case do
       after
         @ex_unit_describe nil
         Module.delete_attribute(__MODULE__, :describetag)
+
```

Comment:
I think `describe/2` should raise if the attribute is set outside the `describe/2` context, in the same way it's done with `@describetag`: `** (RuntimeError) @describetag must be set inside describe/2 blocks`

Right now this is possible:

```elixir
@describe_foo :tortilla
@describe_bar :paella
describe "with attributes outside describe" do
  test "registered subscribe attributes are in context", context do
    assert context.registered.describe_foo == :tortilla
    assert context.registered.describe_bar == [:paella]
  end
end
```

Reply:
We discussed this earlier and it is not possible right now, we don't have the proper API in `Module`. That's why there was a more recent proposal on the mailing list.

Reply:
Oh! I see https://github.com/elixir-lang/elixir/pull/9181#discussion_r298852918 😓